people.json

```diff
@@ -839,5 +839,16 @@
       "LinkedIn": "https://www.linkedin.com/in/bplaat/"
     },
     "contributor": true
+  },
+  {
+    "nick": "Filiph Siitam Sandström",
+    "coordinates": [58.699, 12.453],
+    "links": {
+      "Website": "https://www.sweetsideofsweden.com/",
+      "Company": "https://nordcom.io/",
```

Comment:
Listing a company feels out of place for this list :)

Reply:
Yeah, LinkedIn makes way more sense to list! ☺️